src/app/data-portal/data-portal-details/data-portal-details.component.html

```diff
@@ -476,6 +456,193 @@ <h5 class="card-title rounded mx-auto d-block">{{gen.citeURL}}</h5>
                        style="background-color: limegreen; color: black; margin-top: 5px">
                       Read the GoaT info in full here</a>
                   </mat-tab>
+            <mat-tab label="Symbionts"  *ngIf="organismData?.symbionts_records && organismData?.symbionts_records.length" style="width:100%">
```

Comment:
do we need to have style element here? Would it be better to move it under css? And same thing for other styles tags as well

Reply:
Also check page width, this line goes over the limit